app/templates/detail.hbs

```diff
@@ -1,243 +1,250 @@
 {{page-title this.model.agendaItem.title}}
-<main id="route-detail" class="overflow-y-scroll au-u-flex au-u-margin-bottom-large">
-
-  <article
-    id="detail-left-section"
-    class="au-u-flex au-u-flex--column au-u-4-6"
-  >
-    <AuCard
-      @shadow={{true}}
-      class="color-top-border au-u-flex au-c-card au-u-margin-left-large au-u-margin-right-large au-u-padding"
-      as |c|
-    >
-      <c.header>
-
-        <AuLink
-          @route="detail"
-          class="no-text-decoration au-u-h3 au-u-medium"
-        >
-          {{#if this.model.agendaItem.title}}
-            {{this.model.agendaItem.title}}
+<AuBodyContainer @scroll={{true}} id="route-detail">
+  <AuToolbar @size="medium" @skin="tint" @border="bottom" as |Group|>
+    <Group>
+      <ul class="au-c-list-horizontal au-c-list-horizontal--small">
+        <li class="au-c-list-horizontal__item">
+          <AuLink @route="agenda-items" @icon="arrow-left">
+            Lijst
+          </AuLink>
+        </li>
+        {{#if this.model.agendaItem.title}}
+        <li class="au-c-list-horizontal__item">
+          <span class="u-truncate">{{this.model.agendaItem.title}}</span>
+        </li>
+        {{/if}}
+      </ul>
+    </Group>
+  </AuToolbar>
+  <AuBodyContainer @scroll={{true}}>
+    <div class="au-o-box">
+      <div class="au-o-grid au-o-grid--small">
+        <div class="au-o-grid__item au-u-hidden-from@small">
+          {{#if this.model.agendaItem.handledBy.resolutions}}
+            <AuAlert @skin="success" @icon="sign" @title="Dit werd behandeld" class="au-u-margin-bottom-none" />
+          {{else}}
+            <AuAlert @skin="warning" @icon="three-dots" @title="Dit werd nog niet behandeld" class="au-u-margin-bottom-none" />
           {{/if}}
-
-        </AuLink>
-
-      </c.header>
-      <c.content class="content" style="min-height: 80rem;" {{!template-lint-disable no-inline-styles}}>
-
-        <section
-          class="au-u-flex au-u-muted au-u-margin-top-tiny au-u-u-flex--vertical-center"
-        >
-          <AuIcon class="au-u-margin-right-small" @size="large" @icon="check" />
-          <p>{{this.model.agendaItem.session.dateFormatted}}</p>
-        </section>
-        <section
-          class="au-u-flex au-u-muted au-u-margin-top-tiny au-u-u-flex--vertical-center"
+        </div>
+        <article
+          id="detail-left-section"
+          class="au-o-grid__item au-u-4-6@small"
```

Comment:
This might a bit to small on tablet portrait mode? Maybe `@medium` would be better?